include/fmt/format.h

```diff
@@ -688,6 +688,7 @@ using memory_buffer = basic_memory_buffer<char>;
 using wmemory_buffer = basic_memory_buffer<wchar_t>;
 
 /** A formatting error such as invalid format string. */
+FMT_CLASS_API
 class FMT_API format_error : public std::runtime_error {
```

Comment:
Can we define `FMT_CLASS_API` to include `FMT_API` and simplify this as follows:

```c++
class FMT_CLASS_API format_error : public std::runtime_error {
```
?

Reply:
Nevermind, re-reading the PR description I realized that it's not possible.